### DISCUSSION TITLE
自定义Input组件中,传入组件的onChange,其onChange接收的参数我理解应该就是传入组件中的value,但是我发现onChange(e.target.value)和onChange(e)能达到相同的结果,这是为什么?

### DISCUSSION THREAD

**Discussion by LastStranger** [ant-design/ant-design#37077] [General]
### Reproduction link

[https://github.com/ant-design/ant-design/](https://github.com/ant-design/ant-design/)

### Steps to reproduce

如上

### What is expected?

onChange(e),form.getFieldsValue应该的到的是类似对象的结构, 结果是字符串, 这是antd内部做了某种转换吗?

### What is actually happening?

如上

| Environment | Info |
| --- | --- |
| antd | 4.22.5 |
| React | 18 |
| System | Mac |
| Browser | Chrome91 |

**Comment by zombieJ** [comment]
对：
https://github.com/react-component/field-form/blob/084e2931cbd71e345ef48bb27751a8cf04b54e11/src/utils/valueUtil.ts#L122

**Reply by wangmeijian** [reply]
简单粗暴
